module-apps/CMakeLists.txt

```diff
@@ -84,9 +74,9 @@ endif()
 target_link_libraries(${PROJECT_NAME}
     PRIVATE
         app
-        application-calendar
-        application-messages
-        application-settings
+        $<$<BOOL:${ENABLE_APP_CALENDAR}>:application-calendar>
+        $<$<BOOL:${ENABLE_APP_MESSAGES}>:application-messages>
+        $<$<BOOL:${ENABLE_APP_SETTINGS}>:application-settings>
```

Comment:
I wonder if these apps might be removed from module-apps altogether?

Reply:
It's actually one of many things I wanted to confirm while checking. Unfortunately no :(
I'll actually get back to you on the slack with my thoughts on that and some interface librarisation. Maybe there is some approach we could use from other projects.